packages/components/src/internal/components/picklist/PicklistCreationMenuItem.tsx

```diff
@@ -27,17 +28,26 @@ export const PicklistCreationMenuItem: FC<Props> = props => {
 
     const onClick = useCallback(() => {
         setShowModal(true);
-    }, []);
+    }, [queryModel, sampleIds]);
```

Comment:
these aren't used in the useCallback, so maybe a comment would be useful as to why they were added

Reply:
Oh, well, they were added because they were mistakenly used.  Now they have been removed.